plugins/metrics/src/main/java/org/apache/cloudstack/response/HostMetricsResponse.java

```diff
@@ -43,6 +44,10 @@
     @Param(description = "the total cpu allocated in Ghz")
     private String cpuAllocated;
 
+    @SerializedName("cpuallocatedghzwithoverprovisioning")
+    @Param(description = "the total cpu allocated in Ghz after applying the cpu.overprovisioning.factor", since="4.15.1")
+    private String cpuAllocatedWithOverprovisioning;
```

Comment:
I think allocated cpu metric should involve overprovisioning (if the UI is not doing that); if the overprovisioning factor is not being considered is the cpu alllocated same as usage?